README.md

```diff
@@ -284,6 +284,12 @@ A number of options are available to allow you to customise the SDK:
   smsNumberCountryCode: 'US'
   ```
 
+- **`applicantDetails {Object} optional`**
+  Some applicant details can be specified ahead of time, so that the applicant doesn't need to fill them in themselves.
```

Comment:
let's use the term user instead of applicant in the desc as we tend to refer to them as users in the rest of the readme